docs/_resources/contributors.md

```diff
@@ -0,0 +1,124 @@
+  - [Working on existing bugs](#1-working-on-existing-bugs)
+  - [Suggesting and owning a new feature, fixing a bug, or suggesting a change for an existing Tool](#2-suggesting-and-owning-a-new-feature-fixing-a-bug-or-suggesting-a-change-for-an-existing-tool)
+  - [Submitting a proposal for a new tool](#3-submitting-a-proposal-for-a-new-tool)
+  - [Code review](#4-code-review)
+  - [Answering questions](#5-answering-questions)
+  - [Documentation update](#6-documentation-update)
+
+</details>
+
+---
+
+## 📝 FinOps Toolkit Contribution Guide
+
+Welcome to the FinOps Toolkit project! As an open-source initiative, our success hinges on the collective effort of the community. Your contributions play a vital role in enhancing and expanding the toolkit's capabilities, making it more valuable and effective.
+
+There are three primary ways you can contribute to the success of the project:
+
+1. 🐛 **Working on existing bugs:** You can assist by identifying and addressing existing bugs listed in our repository. 
+2. 🛠️ **Suggesting and owning a new feature, fixing an existing bug, or suggesting a change for an existing tool:** This involves proposing new features, fixing bugs, or suggesting changes for existing tools. 
+3. 💡 **Submitting a proposal for a new tool:** If you've developed a FinOps-related tool and want it to be part of the FinOps Toolkit, you can submit a proposal for consideration.
+4. 👀 **Code review:** Code review is a critical aspect of maintaining code quality and ensuring the reliability and functionality of the project. By reviewing existing pull requests (PRs) suggested by community members, you can contribute significantly to the project's success without necessarily creating new code.
+5. 💬 **Answering questions:** Being actively involved in answering questions and providing assistance to community members is an invaluable contribution to the project.
+6. 📚 **Documentation update:** Documentation serves as a crucial resource for users, developers, and contributors alike. Keeping documentation accurate, comprehensive, and up-to-date is essential for the success of the toolkit.
+
+We appreciate your interest in contributing to the FinOps Toolkit project!
+
+### 🐛 1. Working on existing bugs
+
+If you're eager to contribute but unsure where to begin, a great starting point is to review the list of existing feedback and identify issues that align with your skills and interests. 
+
+1. Look through the list of existing bugs in the [Issues tab](https://github.com/finopsfoundation/FinOpsToolkit/issues).
+2. Assign yourself to the bug you want to work on.
+
+If this is the first time you are working on the FinOps Toolkit repository, make sure to review our [Get Started Wiki.](https://github.com/microsoft/finops-toolkit/wiki#-test-and-verify)
+
+Once you are assigned to the bug, GitHub will automatically assign the tag "Community Contributor" to inform the team responsible for that tool of your contribution. 

```

Comment:
Is this something you're proposing? It doesn't do this today.

Reply:
Yes, I was hoping we could add a GitHub policy to assign this tag. I'm thinking about tracking the work done by the community.
 I will remove it so we can merge this PR but add to our backlog